govc vm.clone SIGSEGV

govc vm.clone SIGSEGV's if target VM exists

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xd4f690]

goroutine 1 [running]:
github.com/vmware/govmomi/govc/vm.(*clone).cloneVM(0xc0001d2000, 0x13ae500, 0xc00003a0b8, 0x0, 0x0, 0x0)
        /home/vmware/src/github.com/vmware/govmomi/govc/vm/clone.go:443 +0x1410
github.com/vmware/govmomi/govc/vm.(*clone).Run(0xc0001d2000, 0x13ae500, 0xc00003a0b8, 0xc00007c7e0, 0x0, 0x0)
        /home/vmware/src/github.com/vmware/govmomi/govc/vm/clone.go:238 +0x24c
github.com/vmware/govmomi/govc/cli.Run(0xc0000300c0, 0xa, 0xa, 0xc000100058)
        /home/vmware/src/github.com/vmware/govmomi/govc/cli/command.go:165 +0x5de
main.main()
        /home/vmware/src/github.com/vmware/govmomi/govc/main.go:99 +0x64

$govc version
govc 0.23.0

--- a/govc/test/vm.bats
+++ b/govc/test/vm.bats
@@ -709,6 +709,9 @@ load test_helper
   run govc vm.clone -cluster DC0_C0 -vm "$vm" "$clone"
   assert_success
 
+  run govc vm.clone -cluster DC0_C0 -vm "$vm" "$clone"
+  assert_failure # already exists
+
   run govc vm.clone -force -vm "$vm" "$clone"
   assert_success # clone vm with the same name
 }

--- a/govc/vm/clone.go
+++ b/govc/vm/clone.go
@@ -440,7 +440,7 @@ func (cmd *clone) cloneVM(ctx context.Context) (*object.VirtualMachine, error) {
 
 		_, err := datastore.Stat(ctx, vmxPath)
 		if err == nil {
-			dsPath := cmd.Datastore.Path(vmxPath)
+			dsPath := datastore.Path(vmxPath)
 			return nil, fmt.Errorf("file %s already exists", dsPath)
 		}
 	}

